apps/nextjs-website/src/components/Form.tsx

```diff
@@ -1,13 +1,36 @@
 'use client';
-import MarkdownRenderer from './MarkdownRenderer';
 import { Form as FormRC } from '@react-components/components';
 import { FormProps } from '@react-components/types';
 import { FormSection } from '@/lib/fetch/types/PageSection';
 import { FormData } from '@react-components/types/Form/Form.types';
 
-const makeFormProps = ({ privacyText, ...rest }: FormSection): FormProps => ({
+const makeFormProps = ({
+  title,
+  subtitle,
+  privacyLinkRecaptchaPolicy,
+  privacyLinkTextRecaptchaPolicy,
+  privacyLinkRecaptchaTerms,
+  privacyLinkTextRecaptchaTerms,
+  formCategories,
+  theme,
+  ctaButtons,
+  ...rest
+}: FormSection): FormProps => ({
+  title,
+  subtitle,
+  privacyLinkRecaptchaPolicy,
+  privacyLinkTextRecaptchaPolicy,
+  privacyLinkRecaptchaTerms,
+  privacyLinkTextRecaptchaTerms,
+  formCategories: formCategories.map(({ label, additionalLabel }) => ({
+    label,
+    additionalLabel: additionalLabel ?? '',
+  })),
+  theme,
+  ctaButtons: ctaButtons.map(({ ...ctaBtn }) => ({
+    ...ctaBtn,
+  })),
```

Comment:
```suggestion
const makeFormProps = ({
  formCategories,
  ...rest
}: FormSection): FormProps => ({
  formCategories: formCategories.map(({ label, additionalLabel }) => ({
    label,
    additionalLabel: additionalLabel ?? '',
  })),
```
Removing unchanged parameters, condensing them into `...rest`